src/synthesizer/line.py

```diff
@@ -620,6 +632,9 @@ def __init__(
                 " passed, or an arbitrary number of Lines to combine"
             )
 
+        # save line_id
+        self.line_id = line_id
```

Comment:
Why have you introduced this? `self.id` contains the line ID and is set in the private constructor methods.

Reply:
Argh... didn't see that. Removed.